catalog/transformer/catalog-transformer-microsoft/src/main/java/ddf/catalog/transformer/input/MicrosoftTransformer.java

```diff
@@ -0,0 +1,296 @@
+                BufferedImage bufferedImage = new BufferedImage(image.getWidth(null),
+                        image.getHeight(null), BufferedImage.TYPE_INT_RGB);
+                Graphics2D graphics = bufferedImage.createGraphics();
+                graphics.drawImage(image, null, null);
+                graphics.dispose();
+
+                BufferedImage thumb = Scalr.resize(bufferedImage, 200);
+
+                try (ByteArrayOutputStream out = new ByteArrayOutputStream()) {
+                    ImageIO.write(thumb, "jpeg", out);
+
+                    byte[] thumbBytes = out.toByteArray();
+                    metacard.setAttribute(new AttributeImpl(Metacard.THUMBNAIL, thumbBytes));
+                }
+            } else {
+                LOGGER.warn("Unable to read image from input stream to create thumbnail.");
+            }
+        } catch (Exception e) {
+            LOGGER.warn("Unable to read image from input stream to create thumbnail.", e);
+        }
+    }
+
+    private String transformToXml(String xhtml) {
+        LOGGER.debug("Transforming xhtml to xml.");
+        try {
+            Writer xml = new StringWriter();
+            Transformer transformer = templates.newTransformer();
+            transformer.transform(new StreamSource(new StringReader(xhtml)), new StreamResult(xml));
+            return xml.toString();
+        } catch (TransformerException e) {
+            LOGGER.warn("Unable to transform metadata from XHTML to XML.", e);
+            return xhtml;
+        }
+    }
+
+    /*
+        XML handler that outputs empty body.
+     */
+    private static class EmptyBodyXMLHandler extends ToXMLContentHandler {
+        private boolean bodyStartTag = false;
+
+        private boolean bodyEndTag = false;
+
+        private boolean insideBodyTag() {
+            return bodyStartTag && !bodyEndTag ? true : false;
+        }
+
+        @Override
+        public void characters(char[] ch, int start, int length) throws SAXException {
+            if (!this.insideBodyTag()) {
+                super.characters(ch, start, length);
+            }
+        }
+
+        @Override
+        public void startElement(String uri, String localName, String qName, Attributes atts)
+                throws SAXException {
+            if (!this.insideBodyTag()) {
+                super.startElement(uri, localName, qName, atts);
+            }
+            /*
+                - once found, stop checking.
+                - check after, so empty body tag is output
+             */
+            if (!bodyStartTag) {
+                bodyStartTag = "body".equals(localName);
+            }
+        }
+
+        @Override
+        public void endElement(String uri, String localName, String qName) throws SAXException {
+            /*
+                - once found, stop checking.
+                - check before, so empty body tag is output
+             */
+            if (!bodyEndTag) {
+                bodyEndTag = "body".equals(localName);
```

Comment:
Should this string comparison be case-insensitive?

Also, since "body" is used twice, you may want to make it a static final constant.